### ISSUE TITLE
Versions pre 4.0.1 do not work

### ISSUE THREAD

**Issue by thedumbtechguy** [simolus3/sqlite3.dart#33]
Unable to use previous versions of the package which means I am forced to upgrade to flutter 2 which we currently are not ready for.

![image](https://user-images.githubusercontent.com/923493/117805142-26023380-b248-11eb-8cad-87dc3640dafa.png)

**Comment by thedumbtechguy** [comment]
Perfect. Thanks

**Comment by rohan20** [comment]
Hi @simolus3 , just for some clarification: This means that all version below `^0.4.0+1` might not work, correct? I get the 403 Forbidden error too just like OP. 

![image](https://user-images.githubusercontent.com/8232573/118956540-fba72900-b95f-11eb-8d25-ba287f37b914.png)

Upgrading to `sqlite3_flutter_libs: ^0.4.2` works 👍

**Comment by simolus3** [comment]
> This means that any version below `^0.4.0+1` might not work, correct?

Yes, exactly. Versions older than `0.4.0` were using bintray which has now shut down. Version `0.4.0` itself uses maven central but I messed up getting the configuration right, so `0.4.0+1` is the first good version.
